crates/topos-p2p/src/behaviour/discovery.rs

```diff
@@ -80,7 +80,10 @@ impl DiscoveryBehaviour {
             next_bootstrap_query: if known_peers.is_empty() {
                 None
             } else {
-                Some(Box::pin(tokio::time::interval(config.bootstrap_interval)))
+                let mut interval = tokio::time::interval(config.bootstrap_interval);
+                interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Delay);
```

Comment:
So the default is `Burst` right? (ref for others: https://docs.rs/tokio/latest/tokio/time/enum.MissedTickBehavior.html#variants)

Reply:
yes it is